tcms/testruns/views.py

```diff
@@ -861,8 +851,6 @@ def get_context_data(self, **kwargs):
             'test_case_run_bugs': bug_ids,
             'mode_stats': mode_stats,
             'summary_stats': summary_stats,
-            'jira_url': jira_url,
-            'bugzilla_url': bugzilla_url
```

Comment:
Both of these should not be removed. IMO, which bug trackers are displayed depends on what bug trackers are defined in the database. Displaying links linking to bugs in the report page is required and a must-have feature in Nitrate. Instead of deleting jira_url and bugzilla_url, it would be better display them by consulting the database. This is the same as adding a issue ID in a test run page.